templates/google-built-opentelemetry-collector/goo/maint.ps1.go.tmpl

```diff
@@ -0,0 +1,49 @@
+# Copyright 2025, Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Requires -Version 3.0
+
+Param(
+    [Parameter(Mandatory=$true)][string]$InstallDir,
+    [Parameter(Mandatory=$true)][ValidateSet('install','uninstall')][string]$Action
+)
+
+$ErrorActionPreference = 'Stop'
+
+$envFromMatch = {
+  Param($match)
+  (Get-ChildItem -Path Env: | `
+     Where-Object -Property Name -eq $match.Groups[1].Value).Value
+}
+$InstallDir = [regex]::Replace($InstallDir,'^<([^>]+)>',$envFromMatch)
+
+$configFilePath = "$InstallDir\config.yaml"
+
+if ($Action -eq "install") {
+    if (-not(Test-Path -Path $configFilePath -PathType Leaf)) {
+         try {
+             New-Item -ItemType directory -Path "$InstallDir\config" -Force
```

Comment:
With `$configFilePath = "$InstallDir\config.yaml"` I think we don't need to create `$InstallDir\config`

Reply:
Fixed in [85b9241](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/297/commits/85b92417cb1b01cac3cdc5b3bafe2f7e4a9fce4d)